ods-setup/setup-ods-project.sh

```diff
@@ -69,6 +69,10 @@ if ! oc adm policy add-cluster-role-to-user self-provisioner system:serviceaccou
   exit 1
 fi
 
+# Allow Jenkins serviceaccount to label namespaces
+oc create clusterrole namespace-labeler --verb=patch --resource=namespace
```

Comment:
I am super worried about this ... can we please try this in the NON ci first ... because that means (likely) that the jenkins user running this needs sys admin rights ... @stitakis

Reply:
The requirement for this fix to work is:
_the oc user used to install ods must have the rights to be able to create the cluster role, otherwise it fails._
In a `odsbox` worked for me because the oc user is `system:admin` and has the rights to create a cluster role. I don't know which permissions has the oc user used to install ods in other clusters. This needs to be verified. 

Regarding the ods jenkins service account...
...I don't know if it needs sys admin rights.
This fix just creates a new cluster role `namespace-labeler` and assign it to ods jenkins service account, so that it gets (additionally to the verbs `get, list, watch`) the verb `patch` on resource `namespace`. This permissions the ods jenkins service account to add labels to namespaces. 

Does this shed some light on this?

Reply:
@clemensutschig to fix the build, I suggest we revert this change https://github.com/opendevstack/ods-core/pull/946 until we have more clarity on this

Reply:
@clemensutschig we tried to create a clusterrole in a corporate oc cluster with the ods installation oc user. It didn't work. So, I'll cancel this PR and create a new one to revert #946.

Reply:
@stitakis @clemensutschig As far as I know, in our on-prem clusters, we are not given any user with permissions to create cluster-scoped resources, e.g. `clusterrole`. 

Instead, we can try creating a local role for a project and then bind it to the Jenkins sa.

```
oc create role namespace-labeler --verb=patch --resource=namespace -n <project>
oc adm policy add-role-to-user namespace-labeler system:serviceaccount:${NAMESPACE}:jenkins -n <project>
````

Reply:
@felipecruz91 it should work, have you tried this in a odsbox and also in your on-prem clusters?